Remora.Rest/Json/DataObjectConverter.cs

```diff
@@ -109,28 +125,26 @@ public DataObjectConverter()
             .ToDictionary(t => t, Activator.CreateInstance);
     }
 
-    private InstancePropertyGetter CreatePropertyGetter
+    private static DTOPropertyWriter CreatePropertyWriter
     (
         PropertyInfo property,
-        InterfaceMapping interfaceMap,
-        PropertyInfo[] interfaceProperties
+        InterfaceMapping interfaceMap
     )
     {
-        var interfaceGetterIndex = Array.IndexOf(interfaceMap.TargetMethods, property.GetGetMethod());
+        var getterMethod = property.GetGetMethod();
+        if (getterMethod is null)
+        {
+            throw new InvalidOperationException($"The property {property.Name} has no public getter");
```

Comment:
Perhaps it'd be better to simply skip over properties with non-public getters? I could see them being useful for some models, though I suppose we can add support later. This might cause issues with explicit interface implementations as well, but I'll need to dig into that.

Reply:
I don't think there should be any. I am not even sure what the previous behavior here was _supposed_ to be, and it looks like it just threw an exception then too.

Reply:
That's fine, then.